_schemaV2.graphql

```diff
@@ -11783,7 +11783,34 @@ type Partner implements Node {
   ): LocationConnection
   meta: PartnerMeta
   name: String
+  partnerArtistsSearchConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    page: Int = 1
+    query: String!
+    size: Int = 10
+  ): partnerArtistsSearchConnection
+  partnerArtworksSearchConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+    page: Int = 1
+    query: String!
+    size: Int = 10
+  ): partnerArtworksSearchConnection
```

Comment:
Same comment here and below (as above). If we already have a connection type that fits (`ArtworksConnection`, `ShowsConnection`), we should be declaring new fields of that type, rather than creating new duplicitous types.

Reply:
I thought this would work and agree the type feels unnecessary but then I try to reuse the other types I get 
`Error: Schema must contain uniquely named types but contains multiple types named "ShowConnection".` 
Putting new fields on the type would work but see comment below on that.

Reply:
There must be something quirky going on, you should be able to reuse types? I can play with it locally or maybe we can pair.